biomodels/.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$/.." vcs="Git" />
+  </component>
+</project>
```

Comment:
I think this is an IntelliJ-specific file. Should it be in github? Same for `biomodels/.idea/compiler.xml` (https://github.com/reactome/data-release-pipeline/pull/96/files#diff-09e35e8ccefda43d436317c68f659dd4)

Reply:
Hmm, past Justin must have included this accidentally.